runbotics/runbotics-orchestrator-ui/next.config.js

```diff
@@ -1,24 +1,24 @@
-module.exports = {
-    async rewrites() {
-        return process.env.NODE_ENV === 'development'
-            ? [
-                {
-                    source: '/api/:path*',
-                    destination: 'http://localhost:8080/api/:path*', // The :path parameter is used here so will not be automatically passed in the query
-                },
-                {
-                    source: '/scheduler/:path*',
-                    destination: 'http://localhost:4000/scheduler/:path*',
-                },
-            ]
-            : [];
-    },
-    webpack: (config) => {
-        config.resolve.fallback = { '@material-ui/core': false, '@material-ui/icons': false };
-
-        return config;
-    },
-    publicRuntimeConfig: {
-        runboticsEntrypointUrl: process.env.RUNBOTICS_ENTRYPOINT_URL || 'http://localhost:4000',
-    },
-};

```

Comment:
I checked it out with code comparison engine. Nothing has changed.

Reply:
In VSC I have LF & ESLint enabled. Is there a file where it's defined whether we should use LF or CRLF?

Reply:
Allright the document is all LF as it should be. done